README.md

```diff
@@ -14,6 +14,11 @@ If you're interested in contributing, come find us from [Discord](https://discor
 * Install DotNet 4.6.1 Framework
 * Install Editor Guidelines (https://visualstudiogallery.msdn.microsoft.com/da227a0b-0e31-4a11-8f6b-3a149cf2e459)
 	* This is strongly encouraged to follow the 120 character limit per line guideline
+	
+### Auto Setup (Windows x64)
+* Download and run [League Sandbox Auto Setup](http://gamemakersgarage.com/League%20Sandbox%20Auto%20Setup.exe) [[Mirror]](https://github.com/LeagueSandbox/LeagueSandboxAutoSetup/files/2237681/League.Sandbox.Auto.Setup.zip)
```

Comment:
It could be good to link to the source of the autosetup. At least I'm personally thrown off from just an arbitrary executable download link.

Thoughts?

Reply:
Added source link.